### ISSUE TITLE
Modify "notify_tag_added" mailer to include tag variants like "questions:____"

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3868]
This section triggers email notifications for tags, like `wetlands`, if you follow that tag (https://publiclab.org/tags)

https://github.com/publiclab/plots2/blob/ec0b0f9df325097eb30504b686f27632021ae28c/app/mailers/subscription_mailer.rb#L29-L55

however, @steviepubliclab and I think it'd be great if it ALSO triggered for some variants of the initial tag. So for `wetlands` subscribers, these should also trigger a notification:

* `question:wetlands`
* `activity:wetlands`
* anything ending in `____:wetlands` 

We could add these notifications by searching for a colon; this will be a bit complex:

```ruby
given_tags = node.tags.reject { |t| t == tag } # this collects tags already on the node, except the current tag
users_to_email = tag.followers_who_dont_follow_tags(given_tags) # who is NOT already notified
# add normal tag version too if it exists:
simple_tag = tag.name.split(':').second
if tag.name.includes?(':') && Tag.find_by(name: simple_tag)
  # now add followers of the normal tag who are NOT already notified
  users_to_email += Tag.find_by(name: simple_tag).followers_who_dont_follow_tags(given_tags)
end
```

I think the ideal is to **create a test** in https://github.com/publiclab/plots2/blob/master/test/unit/subscription_mailer_test.rb where we subscribe a user to the simple tag like `simpletag`, then check that they get a notification when a post goes up for `question:simpletag`. 

This is a bit complex, but we would love some help with it!

**Comment by faithngetich** [comment]
@jywarren I'd like to help work on this issue.

**Comment by jywarren** [comment]
That would be great, thank you!

**Comment by jywarren** [comment]
This could be implemented with a similar system to #6367, actually!
